src/review/logic.py

```diff
@@ -88,9 +205,146 @@ def handle_review_file(_file, content_type, review_assignment, kind):
         stage_uploaded=1,
         kind=kind,
-        owner=review_assignment.user,
+        owner=review_assignment.management_editor,
     )
     new_file.save()
-    review_assignment.files.add(new_file)
 
-    return _path
+    if editorial:
+        review_assignment.editorial_board_files.add(new_file)
+    else:
+        review_assignment.publication_committee_files.add(new_file)
+
+    return path
+
+
+def render_choices(choices):
+    c_split = choices.split('|')
+    return [(choice.capitalize(), choice) for choice in c_split]
+
+
+def create_completed_review_form(submission, review_id):
+    document = Document()
+    document.add_heading(submission.title, 0)
+    review_assignment = get_object_or_404(
+        core_models.ReviewAssignment,
+        pk=review_id,
+    )
+    if review_assignment.review_form:
+        relations = models.FormElementsRelationship.objects.filter(
+            form=review_assignment.review_form,
+        ).order_by('order')
+    else:
+        review_assignment.review_form = submission.review_form
+        review_assignment.save()
+        relations = models.FormElementsRelationship.objects.filter(
+            form=submission.review_form,
+        ).order_by('order')
+
+    if review_assignment.results:
+        data = json.loads(review_assignment.results.data)
+
+        for relation in relations:
+            field_name = core_logic.ascii_encode(relation.element.name)
+            v = data[field_name]
+            document.add_heading(relation.element.name, level=1)
+            text = BeautifulSoup(
+                (v[0]).encode('utf-8'),
+                'html.parser'
+            ).get_text()
+            document.add_paragraph(text).bold = True
+            recommendations = {
+                'accept': 'Accept',
+                'reject': 'Reject',
+                'revisions': 'Revisions Required',
+            }
+
+        document.add_heading("Recommendation", level=1)
+        if recommendations.get(review_assignment.recommendation):
+            document.add_paragraph(
+                recommendations[review_assignment.recommendation]
+            ).italic = True
+        document.add_heading("Competing Interests", level=1)
+        document.add_paragraph(
+            review_assignment.competing_interests
+        ).italic = True
+
+    else:
+        p = document.add_paragraph(
+            'You should complete this form and then use the review assignment '
+            'page to upload it.'
+        )
+
+        for relation in relations:
+
+            if (
+                relation.element.field_type
+                in ['text', 'textarea', 'date', 'email']
+            ):
+                document.add_heading(
+                    relation.element.name + ": _______________________________",
+                    level=1
+                )
+                document.add_paragraph(relation.help_text).italic = True
+
+            if relation.element.field_type in ['select', 'check']:
```

Comment:
Also think this should be an `elif`?

Reply:
I should probably point out: the reason this is showing up as new code is that I refactored it from the `views` to the `logic` module. I will fix these issues anyway as you've gone to the trouble of spotting them.

Reply:
Ah poetic justice :1st_place_medal: